packages/core/src/components/Table/components/mock.ts

```diff
@@ -0,0 +1,26 @@
+export const dataForTableSorting = Array.from({ length: 10 }).map(
+  (_, index) => ({
+    title: `title ${index + 1}`,
+    value: `value ${index + 1}`,
+    description: `description ${index + 1}`,
+  }),
+);
+
+export type DataRow = (typeof dataForTableSorting)[0];
```

Comment:
I'm not quite sure if this file is for Stories only or not. If not, let's use the `./types` file as usual.